ext/droption/droption.h

```diff
@@ -347,12 +363,15 @@ template <typename T> class droption_t : public droption_parser_t
                T minval_, T maxval_,
                std::string desc_short_, std::string desc_long_)
         : droption_parser_t(scope_, name_, desc_short_, desc_long_, 0),
-        value(defval_), defval(defval_), has_range(true),
-        minval(minval_), maxval(maxval_) {}
+        value(defval_), defval(defval_), valsep(DROPTION_DEFAULT_VALUE_SEP),
+        has_range(true), minval(minval_), maxval(maxval_) {}
 
     /** Returns the value of this option. */
     T get_value() const { return value; }
 
+    /** Returns the separator of the value of this option. */
+    std::string get_valsep() const { return valsep; }
```

Comment:
Hmm, `valsep` seems ok for an internal name but for an interface I would suggest `get_value_separator`: WDYT?

Reply:
Agree! Will change to `get_value_separator`